HousingFinanceInterimApi/Dockerfile

```diff
@@ -1,13 +1,25 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0
 
+ARG LBHPACKAGESTOKEN
+ENV LBHPACKAGESTOKEN=$LBHPACKAGESTOKEN
+
 WORKDIR /app
 
 # Copy csproj and restore as distinct layers
-COPY *.csproj ./
-RUN dotnet restore
+
+COPY ./HousingFinanceInterimApi.sln ./
+COPY ./HousingFinanceInterimApi/HousingFinanceInterimApi.csproj ./HousingFinanceInterimApi/
+COPY /NuGet.Config /root/.nuget/NuGet/NuGet.Config
```

Comment:
Line 21 already copies all these files. No need to do this explicitly.

Reply:
Ah, nvm actually ... I didn't notice that API project has a different docker-compose context than the test project. So you'll probably need the explicit nuget file copy.

Reply:
I just copied whats in repairsAPI because I know that works. I will revert most of this back once I get it working